src/Analyzers/Core/CodeFixes/ConflictMarkerResolution/AbstractConflictMarkerCodeFixProvider.cs

```diff
@@ -59,8 +59,8 @@ internal abstract partial class AbstractResolveConflictMarkerCodeFixProvider : C
         /// them if they bring up the lightbulb on a <c>&lt;&lt;&lt;&lt;&lt;&lt;&lt;</c> line, it should run ahead of
         /// normal fix providers else so the user can quickly fix the conflict and move onto the next conflict.
         /// </summary>
-        private protected override CodeActionRequestPriority ComputeRequestPriority()
-            => CodeActionRequestPriority.High;
+        private protected override CodeActionRequestPriorityInternal ComputeRequestPriorityInternal()
+            => CodeActionRequestPriorityInternal.High;
```

Comment:
features that return 'high' use the 'internal' priority system.  For now, only we allow a blessed set to be high pri.  That set is exactly:

1. Add using.
2. Rename
3. Fix formatting
4. Fix git merge-conflicts